test/evergreen.yml

```diff
@@ -4846,6 +4846,7 @@ buildvariants:
     - name: ".pull_request !.pull_request_compilers"
     - name: ".unit_test_long"
     - name: compile
+    - name: s-all
```

Comment:
The task has been tagged as "pull_request", and should have been covered by the above line 4846.

Reply:
Never mind. Just realized you removed the "pull_request" tag on the task. I guess you want the task only runs on the `ubuntu2004` build, which feels reasonable.

Reply:
Yes, that's correct!